.github/workflows/conftest.yaml

```diff
@@ -0,0 +1,16 @@
+name: Validate
+
+on: [push, pull_request]
+
+jobs:
+  conftest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Conftest
+        uses: instrumenta/conftest-action@master
+        with:
+          policy: rego-policy
+          files: basic-dotnet-core/.openshift basic-nginx/.openshift basic-spring-boot-tekton/.openshift basic-spring-boot/.openshift basic-tomcat/.openshift blue-green-spring/.openshift cucumber-selenium-grid/applier/projects cucumber-selenium-grid/applier/templates jenkins-s2i multi-cluster-multi-branch-jee/.openshift multi-cluster-spring-boot/image-mirror-example/.applier/projects multi-cluster-spring-boot/image-mirror-example/.applier/templates multi-cluster-spring-boot/skopeo-example/.applier/projects multi-cluster-spring-boot/skopeo-example/.applier/templates secure-spring-boot/.openshift-applier/templates
```

Comment:
Are these actually expanding the templates? And if not, have we seen that OPA is smart enough to look at the items inside of just the template object?